### PR TITLE
Wipes destination manifests on write. Resolves Issue #16

### PR DESCRIPTION
**Pull request by lprichar**
See Issue #16

### CODE REVIEW

**Comment by jzeferino** [comment]
@lprichar I've changed the base of the merge.
We also use develop as our main develop branch so your feature should derive from develop.
I will need to create the contributing guide for this from now on.

File: src/Cake.AndroidAppManifest/AndroidAppManifest.cs [hunk 1]
```diff
@@ -200,7 +200,7 @@ public void WriteToFile(string fileName)
                 IndentChars = "\t",
                 NewLineChars = "\n"
             };
-            using (var stream = File.OpenWrite(fileName))
+            using (var stream = File.Open(fileName, FileMode.Create))
```
Comment (jzeferino) [inline src/Cake.AndroidAppManifest/AndroidAppManifest.cs]: @lprichar Can you change the `File.Open(string path, System.IO.FileMode mode)` to `File.Open(string path, System.IO.FileMode mode, System.IO.FileAccess access)` so we can specify `FileAccess` as `ReadWrite `? 
See documentation [here](https://docs.microsoft.com/en-us/dotnet/api/system.io.fileaccess?view=netframework-4.7.1) fore more detail.

**Review by jzeferino — CHANGES_REQUESTED** [review]
Please review my comment, and let me know what you think.

**Comment by lprichar** [comment]
You might consider changing your default branch to develop if you'd like PR's to go against there.  Also, I'm pretty sure ReadWrite is the default for all mode's other than Append, but I implemented as requested, no harm in being explicit I guess.

**Comment by jzeferino** [comment]
I don't wan't to change the default branch to master since the master represents the current release and it contains the readme for it.

development can/should have more new features/updates that we don't want to show in the main github page.

We're using gitflow for branch management but it isn't clear yet. I will create a contributing guide with it.

The requested change was for clarity only.

**Pull request merged**